api/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM python:3.9.7-buster
 RUN useradd --create-home service
 
 RUN apt-get update -yqq
-RUN apt-get install -yqq --no-install-recommends binutils libproj-dev gdal-bin
+RUN apt-get install -yqq --no-install-recommends binutils libproj-dev gdal-bin curl
```

Comment:
It's not. I can't explain why but osrd-api was marked as unhealthy since the curl function couldn't be run the container. Installing the package fix it.